packages/python-idna-ssl/python-idna-ssl.spec

```diff
@@ -3,10 +3,10 @@
 
 Name:           python-%{pypi_name}
 Version:        1.1.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 Summary:        Patch ssl.match_hostname for Unicode(idna) domains support
 
-License:        None
```

Comment:
Looks like this package is archived on Github so won't be fixed (nor will we see updates)